Workbooks/Virtual Machines/Virtual Machine Performance/Single Virtual Machine Performance.workbook

```diff
@@ -0,0 +1,515 @@
+{
+  "version": "Notebook/1.0",
+  "isLocked": false,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Single Virtual Machine Performance\n\n*Click on a row to see trends for only that machine*\n---"
+      },
+      "conditionalVisibility": null
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "parameters": [
+          {
+            "id": "8f72709f-725a-41e6-82a2-8d067e38a261",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "isRequired": true,
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.operationalinsights/workspaces": true
+              },
+              "additionalResourceOptions": []
+            },
+            "value": "/subscriptions/f9cf50ed-d814-4b95-b76c-c709845f84c9/resourceGroups/mms-eus/providers/Microsoft.OperationalInsights/workspaces/admdemo"
+          },
+          {
+            "id": "24eb29a6-309b-4b55-aebd-87d3a2057b3f",
+            "version": "KqlParameterItem/1.0",
+            "name": "VirtualMachines",
+            "type": 2,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "Perf\r\n| summarize by Computer\r\n| order by Computer asc\r\n| serialize rank = row_number()\r\n| project Computer, Label = Computer, Selected = iff(rank == 1, true, false)\r\n| union (datatable(Computer:string, Label:string, Selected:boolean)['*', 'All Machines', false])",
+            "crossComponentResources": [
+              "{Workspace}"
+            ],
+            "isHiddenWhenLocked": false,
+            "resourceType": "microsoft.operationalinsights/workspaces",
+            "value": [
+              "admdemo-appsvr",
+              "RHEL75"
+            ]
+          },
+          {
+            "id": "d495f8c1-c6b2-490e-8eb6-43d151bc20c0",
+            "version": "KqlParameterItem/1.0",
+            "name": "Counter",
+            "type": 2,
+            "isRequired": true,
+            "query": "Perf\r\n| where TimeGenerated {TimeRange}\r\n| summarize by CounterName, ObjectName, CounterText = strcat(ObjectName, ' > ', CounterName)\r\n| order by ObjectName asc, CounterText asc\r\n| project Counter = pack('counter', CounterName, 'object', ObjectName), CounterText",
```

Comment:
does this actaully work?  it looks like this parameter depends on time range but actually comes *before* time range? (time range is the *next* parameter?)